Snakefile-proteomics

```diff
@@ -222,6 +222,39 @@ rule final_check_tsv_data_file_not_empty_of_data:
         touch {output}
         """
 
+
+
+rule baseline_markers_proteomics:
+    """
+    Markers calculation for baseline proteomics experiments.
+    """
+    conda: "envs/baseline-markers-genes.yaml"
+    log: "logs/{accession}-baseline_markers.log"
+    resources: mem_mb=get_mem_mb
+    input:
+        starting_check=rules.final_check_tsv_data_file_not_empty_of_data.output,
+        config_xml="{accession}-configuration.xml",
+        expression_file_undecorated="{accession}.tsv.undecorated.aggregated",
+        expression_file="{accession}.tsv",
+        methods_file="{accession}-analysis-methods.tsv"
+    output:
+        markers="{accession}-markers.tsv"
```

Comment:
Can you please keep it in line with tpms and fpkms markers.. its easy for loading script to pick up..

```suggestion
        markers="{accession}-{metric}-markers.tsv"
```

`metric` can be `ppb`

Reply:
Or hard coded

```suggestion
        markers="{accession}-ppb-markers.tsv"
```